### PR TITLE
Replace custom color vars with PatternFly equivalents

### PR DESCRIPTION
**Pull request by rhamilto**

### CODE REVIEW

File: frontend/public/components/_dropdown.scss [hunk 1]
```diff
@@ -32,7 +32,7 @@ $color-bookmarker: #DDD;
   margin: 9px 0;
   overflow: hidden;
   flex-grow: 1;
-  background-color: $color-grey-when-selected;
+  background-color: $color-pf-black-200;
```
Comment (sg00dwin) [inline frontend/public/components/_dropdown.scss]: nit: alpha ?

**Comment by spadgett** [comment]
jenkins rebuild

**Comment by rhamilto** [comment]
jenkins rebuild

**Comment by rhamilto** [comment]
jenkins rebuild

**Comment by spadgett** [comment]
jenkins rebuild

**Comment by rhamilto** [comment]
jenkins rebuild

**Pull request merged**